service/history/configs/config.go

```diff
@@ -115,6 +115,7 @@ type Config struct {
 	TimerProcessorMaxTimeShift                       dynamicconfig.DurationPropertyFn
 	TimerProcessorHistoryArchivalSizeLimit           dynamicconfig.IntPropertyFn
 	TimerProcessorArchivalTimeLimit                  dynamicconfig.DurationPropertyFn
+	RetentionTimerJitterBase                         dynamicconfig.DurationPropertyFn
```

Comment:
nit: jitter duration maybe? I think if this value is X, then retention timer should be evenly distributed from 0 to X? so that the config is more intuitive and easy to understand.

Reply:
Updated comment